kafka/adminapi.go

```diff
@@ -193,28 +193,19 @@ func (c ConfigResource) String() string {
 }
 
 // AlterOperation specifies the operation to perform on the ConfigEntry.
-// One of AlterOperationAdd, AlterOperationSet, AlterOperationDelete.
+// Currently only AlterOperationSet.
 type AlterOperation int
 
 const (
-	// AlterOperationAdd adds the configuration setting.
-	AlterOperationAdd = iota
 	// AlterOperationSet sets/overwrites the configuration setting.
 	AlterOperationSet = iota
-	// AlterOperationDelete deletes the configuration setting,
-	// possibly reverting it to default.
-	AlterOperationDelete = iota
 )
 
 // String returns the human-readable representation of an AlterOperation
 func (o AlterOperation) String() string {
 	switch o {
-	case AlterOperationAdd:
```

Comment:
I think you should take the AlterOperation property out of the ConfigEntry struct altogether.

Reply:
I was tempted to do so too, but Colin's upcoming KIP-339 actually adds a pretty much identical thing, and keeping this type in the Go client makes the ConfigEntry interface forwards compatible.